yarn.lock

```diff
@@ -20955,13 +20955,28 @@ react-router-dom@6.0.0-beta.0:
     prop-types "^15.7.2"
     react-router "6.0.0-beta.0"
 
+react-router-dom@6.2.2:
+  version "6.2.2"
+  resolved "https://registry.npmjs.org/react-router-dom/-/react-router-dom-6.2.2.tgz#f1a2c88365593c76b9612ae80154a13fcb72e442"
+  integrity sha512-AtYEsAST7bDD4dLSQHDnk/qxWLJdad5t1HFa1qJyUrCeGgEuCSw0VB/27ARbF9Fi/W5598ujvJOm3ujUCVzuYQ==
+  dependencies:
+    history "^5.2.0"
+    react-router "6.2.2"
+
 react-router@6.0.0-beta.0:
```

Comment:
@Rugvip @benjdlambert ALL of the references to the old beta have to be gone from the lock file before this can be merged right? Or is there any way that this can be done piecemeal? I think these come from roadie plugins and similar

Reply:
I'd say we need to figure out how to do this piecemeal. Forcing everyone to migrate to v6 stable immediately isn't really something I think we should push out unless there are no other options.